src/components/NewProjectFlow/flow/SelectAccessions.tsx

```diff
@@ -171,6 +174,34 @@ export default function SelectAccessions(props: SelectAccessionsProps): JSX.Elem
     getSearchFields,
   });
 
+  const entitySpecificFilterConfigs: EntitySpecificFilterConfig[] = useMemo(
+    () => [
+      {
+        label: strings.STATUS,
+        initialSelection: filters.statuses || [],
+        filterKey: 'statuses',
+        options: ACCESSION_2_STATES,
+        renderOption: (value: string | number) => `${value}`,
+        pillModifier: (): PillListItemWithEmptyValue[] => {
+          const statuses: AccessionState[] = filters.statuses || [];
+          if (statuses.length === 0) {
+            return [];
+          }
+
+          return [
+            {
+              id: 'statuses',
+              label: strings.STATUS,
+              value: statuses.join(','),
+              emptyValue: [],
+            },
+          ];
+        },
+      },
+    ],
+    [filters]
```

Comment:
also add `activeLocale` from useLocalization.. (unless this already works with locale changes)